### PR TITLE
Update mining-5g.mdx

### PR DESCRIPTION
**Pull request by bobcatminer-admin**
Added several lines to:
1. explain how to access the miner and radio dashboard if using a Bobber 500 instead of a Freedomfi gateway since the URLs are different
2. provide additional CBRS radio troubleshooting resources covering gateway, radio and cellphone connectivity

### CODE REVIEW

**Review by illperipherals — DISMISSED** [review]
on line 39 of mining-5g.mdx, please pluralize or change wording on "You can also find further CBRS radio troubleshooting guide"

**Review by samgutentag — COMMENTED** [review]
made some additional formatting and page structure changes

File: docs/5g-on-helium/mining-5g.mdx [hunk 1]
```diff
@@ -31,11 +31,17 @@ Note that connecting more than one Small Cell Radio will require an additional n
...
+individual 5G Hotspot Makers.
+
+For FreefomFi Gateways refer to the installation guides here:
 
 - [FreedomFi Indoor Install Guide](https://freedomfi.com/freedomfi-indoor-cbrs-small-cell-setup-guide/)
 - [FreedomFi Outdoor Install Guide](https://freedomfi.com/freedomfi-outdoor-cbrs-small-cell-setup-guide/)
 
+For Bobcat500 Gateways refer to the Bobcat Knowledge Center:
+
+- [Bobcat Knowledge Center CBRS Radio Troubleshooting Guide](https://bobcatminer.zendesk.com/hc/en-us/articles/7282743642395-Troubleshooting-Issues-with-the-Indoor-CBRS-Small-Cell-Radio)
```
Comment (meowshka) [inline docs/5g-on-helium/mining-5g.mdx]: Maybe a more generic link will be better? https://bobcatminer.zendesk.com/hc/en-us/articles/4588536001435-Overview

**Review by meowshka — DISMISSED** [review]

File: docs/5g-on-helium/mining-5g.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/5g-on-helium/mining-5g.mdx]: I like it! That links to more guides with lot of good info to share as well

**Review by samgutentag — APPROVED** [review]

**Pull request merged**